One-Electron-Property/Restrained-Electrostatic-Potential/espfit.py

```diff
@@ -0,0 +1,388 @@
+import numpy as np
```

Comment:
For each of these scripts, could you please add appropriate headers according to the conventions [here](https://github.com/psi4/psi4numpy/blob/master/CONTRIBUTING.md#header-styleguide)?